File: go/create_connected_account_request.go
// this file is @generated
package meteroid

type CreateConnectedAccountRequest struct {
	ConnectedOrganizationId string `json:"connected_organization_id"`

	ConnectionType *ConnectionType `json:"connection_type,omitempty"`

	Metadata map[string]any `json:"metadata,omitempty"`

	PlatformCustomerId *CustomerId `json:"platform_customer_id,omitempty"`
}

File: python/meteroid/models/create_connected_account_request.py
# this file is @generated
import dataclasses
import typing as t

from ..serialization import BaseModel
from .connection_type import ConnectionType
from .customer_id import CustomerId


@dataclasses.dataclass
class CreateConnectedAccountRequest(BaseModel):
    connected_organization_id: str

    connection_type: t.Optional[ConnectionType] = None

    metadata: t.Optional[t.Any] = None

    platform_customer_id: t.Optional[CustomerId] = None
